package.json

```diff
@@ -1,14 +1,19 @@
 {
   "name": "delete-tag-and-release",
   "version": "v0.1.1",
-  "main": "index.js",
+  "main": "main.js",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/dev-drprasad/delete-tag-and-release"
   },
   "keywords": [],
+  "scripts": {
+    "test": "jest index.spec.js"
+  },
   "author": "REDDY PRASAD",
   "license": "ISC",
   "homepage": "https://github.com/dev-drprasad/delete-tag-and-release",
-  "dependencies": {}
+  "devDependencies": {
```

Comment:
Shouldn't we also add dependency for `@octokit/action` and `@actions/core` ? If we are using them now?
example from https://github.com/octokit/request-action/blob/main/package.json :
```json
"dependencies": {
    "@actions/core": "^1.2.6",
    "@octokit/action": "^5.0.0",
    "js-yaml": "^4.0.0"
  },
  ```
  
  And what about using `prettier` and `semantic-release` as devDependencies as GitHub Octokit does also?
  Just thinking of other useful things and seeing what others do for testing, release/publishing.

Reply:
No, this isn't using `@actions/core`, `@octokit/action` or `js-yaml` as dependencies so they shouldn't be added.

It would be a good idea to switch to using `@octokit` though, since it provides a wrapper around GitHub's APIs. But that's too big of a change for this PR.

Reply:
OK, looking forward to your next PR for those changes then :-)  I think we'll add you as a core maintainer also, would you mind? :-)

Reply:
Yeah, go for it. I'll still do PRs and what not 'cause what my process but I don't mind helping out. 

And yeah, switching to `@octokit` would make these tests a _lot_ easier so I'll likely do that pretty soon just because I don't like tests that only I can understand. 😆